Authorization/Authorization.py

```
@@ -0,0 +1,49 @@
+import json 
+import base64
+import os
+import requests
+from functools import wraps
+from flask import request,jsonify,make_response
```

Flake8 (7.2.0)

[error] 'flask.make_response' imported but unused

(F401)